### PR TITLE
HDDS-1737. Add Volume check in KeyManager and File Operations.

### PR DESCRIPTION
**Pull request by cxorm**
## NOTICE

Please create an issue in ASF JIRA before opening a pull request,
and you need to set the title of the pull request which starts with
the corresponding JIRA issue number. (e.g. HADOOP-XXXXX. Fix a typo in YYY.)
For more details, please see https://cwiki.apache.org/confluence/display/HADOOP/How+To+Contribute

### CODE REVIEW

File: hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/file/TestOMDirectoryCreateRequest.java [hunk 1]
```diff
@@ -152,6 +152,38 @@ public void testValidateAndUpdateCache() throws Exception {
...
+    OMRequest modifiedOmRequest =
+        omDirectoryCreateRequest.preExecute(ozoneManager);
+
+    omDirectoryCreateRequest = new OMDirectoryCreateRequest(modifiedOmRequest);
+
+    OMClientResponse omClientResponse =
+        omDirectoryCreateRequest.validateAndUpdateCache(ozoneManager, 100L,
+            ozoneManagerDoubleBufferHelper);
+
+    Assert.assertTrue(omClientResponse.getOMResponse().getStatus()
```
Comment (jojochuang) [inline hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/file/TestOMDirectoryCreateRequest.java]: use Assert.assertEquals() instead

File: hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/file/TestOMDirectoryCreateRequest.java [hunk 2]
```diff
@@ -152,6 +152,38 @@ public void testValidateAndUpdateCache() throws Exception {
...
+        == OzoneManagerProtocolProtos.Status.VOLUME_NOT_FOUND);
+
+    // Key should not exist in DB
+    Assert.assertTrue(omMetadataManager.getKeyTable().get(
```
Comment (jojochuang) [inline hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/file/TestOMDirectoryCreateRequest.java]: use assertNull()

Comment (cxorm) [reply hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/file/TestOMDirectoryCreateRequest.java]: Thanks @jojochuang  for the review, I am going to fix it.

**Comment by cxorm** [comment]
Thanks @bharatviswa504 and @jojochuang for the review.
Thanks @elek for the commit.

**Pull request closed without merging**